src/tree/AttachedServersTreeItem.ts

```diff
@@ -42,26 +51,40 @@ export class AttachedServersTreeItem implements IAzureParentTreeItem {
     }
 
     public async loadMoreChildren(_node: IAzureNode, clearCache: boolean): Promise<IAzureTreeItem[]> {
-        return this._attachedServers;
+        return this._attachedAccounts.length > 0 ? this._attachedAccounts : [{
+            contextValue: 'cosmosDBAttachDatabaseAccount',
+            label: 'Attach Database Account...',
+            id: 'cosmosDBAttachDatabaseAccount',
+            commandId: 'cosmosDB.attachDatabaseAccount'
+        }];
     }
 
-    public async attachNewServer(): Promise<void> {
-        const connectionString = await vscode.window.showInputBox({
-            placeHolder: 'mongodb://host:port',
-            prompt: 'Enter the connection string for your database account',
-            ignoreFocusOut: true
-        });
-
-        if (connectionString) {
-            const id = await this.getServerIdFromConnectionString(connectionString);
-            const node = new MongoAccountTreeItem(id, id, connectionString, 'mongoServer');
-            if (this._attachedServers.find(s => s.id === node.id)) {
-                vscode.window.showWarningMessage(`Mongo server '${node.id}' is already attached.`)
-            } else {
-                this._attachedServers.push(node);
-                if (this._keytar) {
-                    await this._keytar.setPassword(this._serviceName, node.id, connectionString);
-                    await this.persistIds();
+    public async attachNewAccount(): Promise<void> {
+        const defaultExperience = <Experience>await vscode.window.showQuickPick(Object.keys(Experience), { placeHolder: "Select a Database Account API...", ignoreFocusOut: true });
+        if (defaultExperience) {
+            let placeholder: string = 'AccountEndpoint=...;AccountKey=...'
+            let validateInput = AttachedAccountsTreeItem.validateDocDBConnectionString;
```

Comment:
I feel placing lines 65,66 in the else block of the following if statement would be clearer. No strong opinions either way though.

Reply:
No strong preference either. Updated to reflect your suggestion